src/chapters/css/exercises.rst

```diff
@@ -0,0 +1,13 @@
+Exercises: CSS
+==============
+
+We have built a website for you to test your CSS knowledge on. Check out the `repl.it <https://repl.it/@launchcode/CSSExercises/>`_!
+
+For the exercises, add the following style rules to ``style.css``:
+
+1. Change the background color to yellow.
+2. Change all paragraph text color to red.
+3. Change all ``h1`` to 36 px font size.
+4. Align all text to the center of the page.
```

Comment:
What do you think about making this step have multiple parts?  I can think of at least three ways to center all the text on the page.  Perhaps we can ask the students to do something similar?

Example:
4. Align all text to the center of the page.
    a. Now find a *different* way to center the text on the page.
    b. Can you find *another* way to center the text on the page?

Reply:
They only learn how to use ``text-align``.

Reply:
Yes, but you can ``text-align`` each tag, ``text-align`` a class, or encase the tags in one ``div`` element and ``text-align`` that.

Reply:
I added an exercise to use a CSS id to align only the headings to the center.